jsk_perception/sample/sample_darknet_yolo_object_detection.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
```

Comment:
@wkentaro Ah, OK, I now understand. This nodelet itself supports also other config and weights in addition to `yolo`, and `darknet_yolo_object_detection.launch` and `sample_....launch` is just one configuration for `yolo` network. You can add other launch files with only change of launch files.